releases/xrpl/2025-02-v1.0.0.md

```diff
@@ -0,0 +1,230 @@
+ENV=xyz
+CHAIN=xyz
+```
+
+An initial XRPL chain config needs to be added to `${ENV}.json` file under `chains` key.
+
+#### Devnet-Amplifier
+
+```bash
+\"$CHAIN\": {
+  "name": "XRPL",
+  "axelarId": \"$CHAIN\",
+  "rpc": "https://s.devnet.rippletest.net:51234",
+  "wssRpc": "wss://s.devnet.rippletest.net:51233",
+  "tokenSymbol": "XRP",
+  "networkType": "testnet",
+  "chainType": "xrpl",
+  "finality": "1",
+  "approxFinalityWaitTime": 1,
+  "explorer": {
+    "name": "XRPL Explorer",
+    "url": "https://devnet.xrpl.org"
+  },
+  "contracts": {}
+}
+```
+
+#### Stagenet / Testnet
+
+```bash
+\"$CHAIN\": {
+  "name": "XRPL",
+  "axelarId": \"$CHAIN\",
+  "rpc": "https://s.altnet.rippletest.net:51234",
+  "wssRpc": "wss://s.altnet.rippletest.net:51233",
+  "tokenSymbol": "XRP",
+  "networkType": "testnet",
+  "chainType": "xrpl",
+  "finality": "1",
+  "approxFinalityWaitTime": 1,
+  "explorer": {
+    "name": "XRPL Explorer",
+    "url": "https://testnet.xrpl.org"
+  },
+  "contracts": {}
+}
+```
+
+#### Mainnet
+
+```bash
+"xrpl": {
+  "name": "XRPL",
+  "axelarId": "xrpl",
+  "rpc": "https://s1.ripple.com:51234",
+  "wssRpc": "wss://s1.ripple.com",
+  "tokenSymbol": "XRP",
+  "networkType": "mainnet",
+  "chainType": "xrpl",
+  "finality": "1",
+  "approxFinalityWaitTime": 1,
+  "explorer": {
+    "name": "XRPL Explorer",
+    "url": "https://livenet.xrpl.org"
+  },
+  "contracts": {}
+}
+```
+
+1. Request tokens from faucet
+
+```bash
+node xrpl/faucet.js
+```
+
+2. Deploy XRPL Multisig account (edge AxelarGateway equivalent)
+
+```bash
+node xrpl/deploy-multisig.js --initial-signers r4LDyA6WoyzJ6ZhQ5NWU4AG1aApBwbSSr4LDyA6WoyzJ6ZhQ5NWU4AG1aApBwbSSk6
+```
+
+3. After creating the XRPL Multisig, ensure that you deploy the [XRPL Amplifier contracts](../cosmwasm/2025-02-XRPL-v1.0.0.md)
+
+4. Set genesis verifier set
+
+```bash
+axelard tx wasm execute $XRPL_MULTISIG_PROVER '"update_verifier_set"' "${ARGS[@]}"
+```
+
+5. Register XRPL Multisig on ITS Hub
+
+ITS hub contract configuration in `$ENV.json` must include the following attributes per chain:
+
+```bash
+"axelar": {
+  "contracts": {
+    ...
+    "InterchainTokenService": {
+      ...
+      \"$CHAIN\": {
+        "maxUintBits": 256,
+        "maxDecimalsWhenTruncating": 255
+      }
+    }
+    ...
+  }
+}
+```
+
+Please refer to `$DEPOSIT_VALUE` and `$RUN_AS_ACCOUNT` from [XRPL Amplifier](../cosmwasm/2025-02-XRPL-v1.0.0.md).
+
+```bash
+node cosmwasm/submit-proposal.js \
+    its-hub-register-chains $CHAIN \
+    -t "Register $CHAIN on ITS Hub" \
+    -d "Register $CHAIN on ITS Hub" \
+    --deposit $DEPOSIT_VALUE \
+    --runAs $RUN_AS_ACCOUNT
+```
+
+6. Set XRPL as trusted chain on EVM ITS. Similarly, set XRPL as a trusted chain for every other non EVM ITS contract
+
+```bash
+# Change `PRIVATE_KEY and `ENV` in `.env` from XRPL to EVM
+node evm/its.js -n all --action setTrustedAddress --trustedChain $CHAIN --trustedAddress hub
+```
+
+7. Register XRP token as admin
```

Comment:
let's specify the salt is supposed to be a 32byte hex

Reply:
Added a comment.